cap-fs-ext/src/dir_ext.rs

```diff
@@ -238,6 +262,24 @@ impl DirExt for cap_std::fs::Dir {
             Err(e) => Err(e),
         }
     }
+
+    #[cfg(not(windows))]
+    #[inline]
+    fn remove_file_or_symlink<P: AsRef<Path>>(&self, path: P) -> io::Result<()> {
+        self.remove_file(path.as_ref())
+    }
+
+    #[cfg(windows)]
+    #[inline]
+    fn remove_file_or_symlink<P: AsRef<Path>>(&self, path: P) -> io::Result<()> {
+        match self.remove_file(path.as_ref()) {
+            Ok(()) => Ok(()),
+            Err(e) => match self.remove_dir(path.as_ref()) {
+                Ok(()) => Ok(()),
+                Err(_) => Err(e),
+            },
+        }
```

Comment:
The unfortunate part here is that we can't avoid removing an empty directory here. But I think I convinced myself that's ok; it's only an empty directory after all. But, we should mention that in the documentation comment.

Reply:
Good point! e6cb016